cmd/crio/main.go

```diff
@@ -296,6 +299,10 @@ func main() {
 			Value: libkpod.DefaultPidsLimit,
 			Usage: "maximum number of processes allowed in a container",
 		},
+		cli.BoolFlag{
+			Name:  "share-pid-namespace",
+			Usage: "using a shared PID namespace for containers in a pod",
```

Comment:
default to use a  spared PID Namespace for containes in a pod (Default is false)

Reply:
Do we want the default to be true?